runtime/process.go

```diff
@@ -260,8 +260,12 @@ func (p *process) Signal(s os.Signal) error {
 // This should only be called on the process with ID "init"
 func (p *process) Start() error {
 	if p.ID() == InitProcessID {
-		_, err := fmt.Fprintf(p.controlPipe, "%d %d %d\n", 2, 0, 0)
-		return err
+		args := p.container.runtimeArgs
```

Comment:
Sync with process. Wait here? Runc init could die in between and this will stay stuck forever in that case.